tailwind.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
 
   theme: {
     extend: {
+      display: ["group-hover"],
```

Comment:
What does this do?

Reply:
Thought it enabled group-hover class to enable hover at the parent level. Removed.

Reply:
This doesn't seem to ever get removed? I thought this was not needed anymore?